django_celery_beat/locale/fr/LC_MESSAGES/django.po

```diff
@@ -229,10 +229,6 @@ msgstr "Horaire"
 msgid "Run the task at clocked time"
 msgstr "Démarre la tâche à l'horaire définie"
 
-#: django_celery_beat/models.py:216 django_celery_beat/models.py:516
```

Comment:
can you point to another one of which it is duplicate?

Reply:
Sure.

https://github.com/celery/django-celery-beat/blob/6172ba4b484cb4173ec7a28a31573ef26877cd8c/django_celery_beat/locale/fr/LC_MESSAGES/django.po#L233

https://github.com/celery/django-celery-beat/blob/6172ba4b484cb4173ec7a28a31573ef26877cd8c/django_celery_beat/locale/fr/LC_MESSAGES/django.po#L471